package.json

```diff
@@ -255,6 +255,7 @@
         "lodash": "^4.17.11",
         "mousetrap": "^1.6.3",
         "portfinder": "^1.0.21",
+        "tcp-port-used": "^1.0.2",
```

Comment:
shouldn't this be dev dependency ?

Reply:
Fixed with https://github.com/ballerina-platform/plugin-vscode/pull/131